### PR TITLE
console log for cx developers when widget id is no longer found in the system

### PR DESCRIPTION
**Pull request by mithunjohny**
# PR Details

### Description

_Include a description of the problem to be solved_

## Solution Proposed

_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria

_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence

_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### Sanity Tests

- [ ] You have tested all changes in Popout mode
- [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
- [x] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y

- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

***Please provide justification if any of the validations has been skipped.***

### CODE REVIEW

File: src/core/OmnichannelErrorCodes.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 enum OmnichannelErrorCodes {
     WidgetUseOutsideOperatingHour = 705,
+    noWidgetPresentErrorCode = 701,
```
Comment (xTEddie) [inline src/core/OmnichannelErrorCodes.ts:3]: WidgetNotFound?

It's already implied that it's an error code based on the name `OmnichannelErrorCodes`. So, `error code` can be removed from the naming.

File: src/OmnichannelChatSDK.ts [hunk 2]
```diff
@@ -2146,6 +2146,10 @@ class OmnichannelChatSDK {
                 this.OCClient = await OCSDKProvider.getSDK(this.omnichannelConfig as IOmnichannelConfiguration, createOcSDKConfiguration(false) as ISDKConfiguration, this.ocSdkLogger as OCSDKLogger);
                 liveChatConfig = await this.OCClient.getChatConfig(this.requestId, bypassCache); // Bubble up error by default to throw ChatConfigRetrievalFailure
             } else {
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                if ((error as any).response?.headers?.errorcode && parseInt((error as any).response.headers.errorcode) === OmnichannelErrorCodes.noWidgetPresentErrorCode) {
+                    console.log("No widget with the given app id is present in the system.");
```
Comment (xTEddie) [inline src/OmnichannelChatSDK.ts:2151]: Should it be a `console.error()` or `console.warn`?

**Review by elopezanaya — APPROVED** [review]

**Review by matan-hershberg-nuance — APPROVED** [review]
